vite.config.ts

```diff
@@ -0,0 +1,8 @@
+import react from "@vitejs/plugin-react";
+import { defineConfig } from "vite";
+import tsconfigPaths from "vite-tsconfig-paths";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react(), tsconfigPaths()],
```

Comment:
**question:** do we need tsconfigPaths? i don't think we have any path aliases in our tsconfig

Reply:
I don't think we do to be honest? I was mainly just following what it says here: https://www.chakra-ui.com/docs/get-started/frameworks/vite#setup-vite-config-paths